public/content/developers/tutorials/using-websockets/index.md

```diff
@@ -242,4 +242,4 @@ curl https://eth-mainnet.alchemyapi.io/v2/your-api-key
 
 ---
 
-[Sign up with Alchemy](https://auth.alchemyapi.io/signup) for free, check out [our documentation](https://docs.alchemyapi.io/), and for the latest news, follow us on [Twitter](https://twitter.com/AlchemyPlatform).
+[Sign up with Alchemy](https://auth.alchemy.com/signup) for free, check out [our documentation](https://www.alchemy.com/docs/), and for the latest news, follow us on [Twitter](https://twitter.com/AlchemyPlatform).
```

Comment:
<img width="1411" height="589" alt="image" src="https://github.com/user-attachments/assets/1e336b89-7916-414e-9b27-f6e9b31a6bd0" />

Looks like this one redirecting to root, will patch

```suggestion
[Sign up with Alchemy](https://auth.alchemy.com) for free, check out [our documentation](https://www.alchemy.com/docs/), and for the latest news, follow us on [Twitter](https://x.com/AlchemyPlatform).
```